addons/stock_landed_costs/tests/test_stock_landed_costs.py

```diff
@@ -217,3 +217,29 @@ def test_aml_account_selection(self):
             landed_cost_aml = bill.invoice_line_ids.filtered(lambda l: l.product_id == self.landed_cost)
             self.assertEqual(bill.state, 'posted', 'Incorrect value with valuation %s' % valuation)
             self.assertEqual(landed_cost_aml.account_id, account, 'Incorrect value with valuation %s' % valuation)
+
+    def test_landed_cost_in_move_line(self):
+        """
+        Tests that a move line created through the catalog gives the right landed cost
+        """
+        landed_cost_product = self.env['product.product'].create({
+            'name': 'Landed Cost Product',
+            'type': 'service',
+            'landed_cost_ok': True,
+        })
```

Comment:
`self.landed_cost.landed_cost_ok = True` is enough (just change it for the rest)